src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -199,6 +205,7 @@ class Task extends React.Component<Props, State> {
         const {
             approverSelectorContacts,
             assigned_to,
+            completion_rule = TASK_COMPLETION_RULE_ALL,
```

Comment:
Can the default value live in `defaultProps`? It helps keep the expected behavior consistent throughout the component.